docs/rules/named-functions-in-promises.md

```diff
@@ -2,6 +2,19 @@
 
 ### Rule name: `named-functions-in-promises`
 
+#### Configuration
+
+```
+ember/named-functions-in-promises: [2, {
+  allowConciseArrow: false,
```

Comment:
That sounds good and would probably make more sense to users. I was trying to reuse the language from MDN regarding ["concise body"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions).